lib/src/replay/mask/posthog_mask_controller.dart

```diff
@@ -67,4 +67,29 @@ class PostHogMaskController {
       return null;
     }
   }
+
+  List<Rect>? getPostHogWidgetWrapperElements() {
+    final BuildContext? context = containerKey.currentContext;
```

Comment:
```suggestion
    final context = containerKey.currentContext;
```

double heck the others too